### DISCUSSION TITLE
$primary-hue in V2

### DISCUSSION THREAD

**Discussion by joeinnes** [picocss/pico#466] [Help]
Hi folks,

In the previous version you could quickly and easily set a very specific hue degree, and an entire palette would be generated.

Although I love the curated palettes in v2, are there any plans to bring this option back? It was very useful for generating quick, branded sites with minimal effort.

Thanks!

**Comment by Leftium** [comment]
If you are talking about https://picocss.com/docs/v1/customization, the v2 version is here:
- Custom primary color: https://github.com/picocss/examples/blob/a1668dca807f821907c46e979c66a0f68e1a5275/v2-sass-customized-design-system/scss/theme/_light.scss#L7
- Using predefined colors: https://picocss.com/docs/version-picker/slate

**Comment by Leftium** [comment]
Or perhaps you were referring to this. You can just copy-paste this SASS and use with the custom primary color example:
- Custom primary color example (don't forget to set `primary-inverse`)
    - [sandbox](https://codesandbox.io/p/devbox/github/picocss/examples/tree/master/v2-sass-customized-design-system?embed=1&file=%2Fscss%2Fmain.scss)
    - [preview](https://gddt65-3000.csb.app/)
    - [code](https://github.com/picocss/examples/tree/master/v2-sass-customized-design-system)
- Copy-paste this SASS: [v1-basic-template/scss/custom/_colors.scss#L15-L25](https://github.com/picocss/examples/blob/a1668dca807f821907c46e979c66a0f68e1a5275/v1-basic-template/scss/custom/_colors.scss#L15-L25)
```
$primary-hue: 180;
$primary-50: hsl($primary-hue, 90%, 89%) !default;
$primary-100: hsl($primary-hue, 88%, 81%) !default;
$primary-200: hsl($primary-hue, 86%, 72%) !default;
$primary-300: hsl($primary-hue, 84%, 63%) !default;
$primary-400: hsl($primary-hue, 82%, 54%) !default;
$primary-500: hsl($primary-hue, 80%, 45%) !default;
$primary-600: hsl($primary-hue, 85%, 36%) !default;
$primary-700: hsl($primary-hue, 90%, 27%) !default;
$primary-800: hsl($primary-hue, 95%, 18%) !default;
$primary-900: hsl($primary-hue, 100%, 10%) !default;
```
